src/test/java/com/marklogic/kafka/connect/source/DslConstraintInjectionTest.java

```diff
@@ -1,61 +1,202 @@
 package com.marklogic.kafka.connect.source;
 
-import org.junit.jupiter.api.Assertions;
+import com.marklogic.client.expression.PlanBuilder;
+import org.apache.kafka.connect.source.SourceRecord;
 import org.junit.jupiter.api.Test;
 
+import java.io.IOException;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
 class DslConstraintInjectionTest extends AbstractIntegrationSourceTest {
-    private static final String constraintColumn = "lucky_number";
-    private static final String constraintValue = "52";
+    private static final String constraintColumn = "ID";
+    private static final String constraintValue = "2";
+    private static final String expectedWhereClause = ".where(op.gt(op.col(\"" + constraintColumn + "\"), \""+ constraintValue + "\"))";
```

Comment:
To make it easier to read these queries, I recommend using single quotes so you don't need ugly escaped double quotes in Java. 

Also, I go back and forth on using variables for strings in tests. The reason is that it makes understanding the test a little harder - i.e. this would be easier to read, even though it duplicates some stuff:

    private static final String easierToReadClause = ".where(op.gt(op.col('ID'), '2'))";

Thoughts on that? It's a balance between duplication of values and readability of the test, there's not really a right answer.